### PR TITLE
Travis: fix Windows itest

### PR DESCRIPTION
**Pull request by guggero**
Not sure if this will have an effect on the itest runs on Travis, just testing.

### CODE REVIEW

**Comment by guggero** [comment]
I updated the PR to decrease the number of concurrent tests on Windows to 2 (from 4).
And I also found a bug in the SCB recovery tests that sometimes lead to the nodes not being able to connect to each other again if they got new P2P ports after restart.

With the current state the Windows build was green on first try.

File: lntest/itest/lnd_channel_backup_test.go [hunk 1]
```diff
@@ -120,10 +120,17 @@ func testChannelBackupRestore(net *lntest.NetworkHarness, t *harnessTest) {
 				// Create a new nodeRestorer that will restore
 				// the node using the Multi backup we just
 				// obtained above.
+				oldPortOpts := func(cfg *lntest.NodeConfig) {
```
Comment (bhandras) [inline lntest/itest/lnd_channel_backup_test.go:123]: nit: to avoid copy-paste you could also create a function returning a `NodeOption`, something like:
```
func copyNodePorts(oldNode *HarnessNode) NodeOption {
    return func(cfg *lntest.NodeConfig) {
		cfg.P2PPort = oldNode.Cfg.P2PPort
		cfg.RPCPort = oldNode.Cfg.RPCPort
		cfg.RESTPort = oldNode.Cfg.RESTPort
		cfg.ProfilePort = oldNode.Cfg.ProfilePort
	}
}
```

**Review by bhandras — APPROVED** [review]
LGTM :+1:

File: .travis.yml [hunk 2]
```diff
@@ -77,7 +77,7 @@ jobs:
 
     - name: Btcd Integration Windows
       script:
-        - make itest-parallel windows=1
+        - make itest-parallel windows=1 ITEST_PARALLELISM=2
```
Comment (halseth) [inline .travis.yml:80]: The change should be accompanied with a comment (or commit msg) with the rationale

File: lntest/itest/lnd_channel_backup_test.go [hunk 3]
```diff
@@ -825,9 +844,12 @@ func testChanRestoreScenario(t *harnessTest, net *lntest.NetworkHarness,
 
 	ctxb := context.Background()
 
-	var nodeArgs []string
+	nodeArgs := []string{
+		"--minbackoff=50ms",
```
Comment (halseth) [inline lntest/itest/lnd_channel_backup_test.go:848]: change should be its own commit?

**Review by halseth — APPROVED** [review]
LGTM, just a few nits re commit style and messages

File: lntest/itest/lnd_channel_backup_test.go (hunk 1 above)
Comment (guggero) [reply lntest/itest/lnd_channel_backup_test.go]: Great idea, done.

File: .travis.yml (hunk 2 above)
Comment (guggero) [reply .travis.yml:82]: You're right, done.

File: lntest/itest/lnd_channel_backup_test.go (hunk 3 above)
Comment (guggero) [reply lntest/itest/lnd_channel_backup_test.go:831]: Fixed.
